docs/domains/Domain.md

```diff
@@ -193,6 +194,24 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | `annotations` | Map | The annotations to be attached to generated resources. |
 | `labels` | Map | The labels to be attached to generated resources. The label names must not start with 'weblogic.'. |
 
+### Model
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `configMapName` | string | WDT config map name |
+| `domainType` | string | WDT domain type: Legal values: WLS, RestrictedJRF, JRF |
+| `encryptionSecret` | [Secret Reference](k8s1.13.5.md#secret-reference) | WDT encryption key pass phrase secret. |
+| `opss` | [Opss](#opss) | Configuration for OPSS based security |
+| `rollbackIfRequireStart` | Boolean | Rollback dynamic changes if the updates requires restart |
+| `useOnlineUpdate` | Boolean | Use online update during lifecycle changes |
+
+### Overrides
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `configMapName` | string | The name of the config map for WebLogic configuration overrides. |
+| `secrets` | array of string | A list of names of the secrets for WebLogic configuration overrides. |
```

Comment:
I'm thinking secrets should be moved up one level as they're shared by both MII and overrides.   Starting discussion in slack.